tests/fixtures/app/yarn-travis/package.json

```diff
@@ -49,9 +50,9 @@
     "ember-source": "~3.27.2",
     "ember-template-lint": "^4.0.0",
     "ember-welcome-page": "^4.0.0",
-    "eslint": "^7.27.0",
+    "eslint": "^8.0.1",
     "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-ember": "^10.4.2",
+    "eslint-plugin-ember": "^10.5.4",
```

Comment:
This doesn't seem to be properly rebased. This is an older version than is on master: https://github.com/ember-cli/ember-cli/blob/1e415adb425fe219f4d8708be64c792be81f7385/blueprints/app/files/package.json#L59

Reply:
this is not the same file, mine is `tests/fixtures/app/yarn-travis/package.json`, your is `ember-cli/blueprints/app/files/package.json`

maybe I should update all `eslint-plugin-ember` to `^11.1.0`？

Reply:
With a few searches and replace, all occurrences are now updated to the `^11.10`.